Add details on oracle spec referring internal oracles

ValentinTrinque opened the pull request:
This has been implemented but not documented.

ValentinTrinque committed:
Add details on oracle spec referring internal oracles

ValentinTrinque committed:
Better example

barnabee commented on protocol/0048-DSRI-data_source_internal.md:
```diff
@@ -51,6 +51,31 @@ on: {
+
+Usage of internal oracle data are specified using properties prefixed with `vegaprotocol.builtin`, on the oracle spec.
+
+Today, only the time-triggered one is implemented through the property name `vegaprotocol.builtin.timestamp`.
```
We need to addjust this text, this is not a time trigger so much as a built in timestamp data source. We should give it a separate section for built in data sources is it behaves functionally differently. I can do this update.

barnabee reviewed (changes requested):
Need to update this still. I will do this.

barnabee committed:
Update spec for built in Vega time source

- clarify this is not the same as time triggered source
- update implmentation details to show that time triggered is not implemented
- add ACs for Vega time source

barnabee reviewed (dismissed):
> We need to addjust this text, this is not a time trigger so much as a built in timestamp data source. We should give it a separate section for built in data sources is it behaves functionally differently. I can do this update. 

I've made these updates, please rereview

gordsport committed:
chore: update AC codes

Mark old ACs for the next milestone and renumber the new acs to avoid conflicts

gordsport committed:
Merge branch 'master' into internal-oracles
